Objects/Converters/ConverterRevit/ConverterRevitShared/AllRevitCategories.cs

```diff
@@ -132,21 +138,31 @@ public IRevitCategoryInfo GetRevitCategoryInfo(string categoryName)
     }
     #endregion
 
+
+
     private IRevitCategoryInfo? GetCategoryInfoForObjectWithExactName(string unformattedCatName)
```

Comment:
These changes a re a bit less relevant as they touch the type mapper, but I thought to alight its behavior too.